frontend/package.json

```diff
@@ -21,6 +21,7 @@
     "nightly": "yarn deploy",
     "dev": "yarn start:proxy:beta",
     "start": "webpack serve --config config/dev.webpack.config.js",
+    "start:k8s": "REMOTE_CONFIG=ci ENVIRONMENT=prod BETA=true PROXY=true yarn start",
```

Comment:
The `REMOTE_CONFIG` should be at least `stage` as `ci` is no longer supported.

Reply:
Interesting, `stage` fails to find the config 🤔 

And `ci` works for some reason... is the config not deployed to `stage`?

> chrome-root.59d856e245e174c085dd.js:2 GET https://prod.foo.redhat.com:1337/beta/config/chrome/fed-modules.json?ts=1643296704912 403 (Forbidden)